server/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -428,13 +428,13 @@ public void testCustomCharFilterWithParameters() throws IOException {
     public void testNormalizerWithIndex() throws IOException {
         AnalyzeAction.Request request = new AnalyzeAction.Request("index");
         request.normalizer("my_normalizer");
-        request.text("ABc");
+        request.text("Wi-fi");
```

Comment:
This is great and absolutely correct. 
It took me a while reading the code though to understand though why this fails without the additions you made in TransportAnalyzeAction. Can you add something along the lines of the following comment to explain that we expect this normalizer to use the "keyword" tokenizer and "lowercase" filter. 
The standard analyzer thats picked by error now also seems to lowercase which is why the test passed with the current test text. Lets make the assumptions around the tokenizer also very clear here.

```suggestion
        // this should be tokenized with the "keyword" tokenizer and then lowercased
        request.text("Wi-fi");
```

Reply:
Hi @cbuescher , from the document of [Normalizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-normalizers.html) , I notice that normalizer do not have a tokenizer and they are only worked on keyword datatype. With the standard analyzer, the text "Wi-fi" will emit "wi" and "fi",  so I think the comment can be as follows:
```suggestion
        // this should be lowercased and only emit a single token
        request.text("Wi-fi");
```

Reply:
Sure, lets go with that. I'll run tests and merge after you commited this version. Thanks a lot.